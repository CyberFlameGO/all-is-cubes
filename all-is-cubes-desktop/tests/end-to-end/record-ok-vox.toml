bin.name = "all-is-cubes"
args = "--no-config-files --template lighting-bench --graphics record -o test-record-lighting-bench.vox"
