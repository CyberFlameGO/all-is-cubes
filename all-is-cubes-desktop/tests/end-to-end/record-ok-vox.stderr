...

Wrote test-record-lighting-bench.vox
